src/vsr/replica.zig

```diff
@@ -8088,9 +8115,10 @@ pub fn ReplicaType(
             assert(self.view_durable_updating());
         }
 
-        fn primary_send_start_view_after_view_change(self: *Self) void {
-            assert(self.status == .normal);
-            assert(self.primary());
+        fn send_start_view(self: *Self) void {
```

Comment:
Imo it is ok to call this `primary_send_start_view`, even though the replica might not be in normal status.